modules/setting/service.go

```diff
@@ -17,6 +17,7 @@ var Service struct {
 	ActiveCodeLives                         int
 	ResetPwdCodeLives                       int
 	RegisterEmailConfirm                    bool
+	RegisterManualConfirm                   bool
```

Comment:
need set the value in ``newService()`` (see https://github.com/go-gitea/gitea/pull/13083/files#diff-0ab2a298fe8b3109330b3679b3ae75683af157d2e1123251f21e4bf6ed0c08efR61 )

Reply:
Sorry, I am a bit lost. The `RegisterManualConfirm` follows the implementation of `RegisterEmailConfirm` and `RegisterEmailConfirm` is not set in `newService()` neither.

Reply:
it's setting in here
https://github.com/go-gitea/gitea/blob/63aeb0a859499623becc1d1e7c8a2ad57439e139/modules/setting/mailer.go#L119

Reply:
Oh, I got it. Should be solved now (https://github.com/go-gitea/gitea/pull/13083/commits/95a3b174d0ae85559f97192ebe7e02c1ee43d9c4#diff-0ab2a298fe8b3109330b3679b3ae75683af157d2e1123251f21e4bf6ed0c08efR67)